scripts/generate_instrumentation_bootstrap.py

```diff
@@ -93,6 +93,14 @@ def main():
                     values=[ast.Str(target_pkg), ast.Str(pkg["requirement"])],
                 )
             )
+        # _instruments_either is an optional field that can be used instead of or in addition to _instruments. While _instruments is a list of dependencies, all of which are expected by the instrumentation, _instruments_either is a list any of which but not all are expected.
```

Comment:
Just a suggestion but I wonder if `instruments_any` would be more appropriate than `instruments_either` since we're matching on one of multiple rather than one of two.